src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let price = 0;
+
+  for (let i = 0; i < days; i++) {
+    price += rent;
+  }
+
+  if (days >= 3 && days < 7) {
+    return price - 20;
```

Comment:
what if our discount changes? (checklist p1)

> [VARIABLES] - use variables for the main values so that you can reuse them and give them descriptive names.